src/pages/basic_data/institution_admin/edit_user/UserFormDialog.tsx

```diff
@@ -68,9 +68,23 @@ export const UserFormDialog = ({ open, onClose, existingUser, existingPerson }:
     meta: { errorMessage: t('feedback.error.get_person') },
     initialData: existingPersonObject,
   });
+  const personEmployments = personQuery.data?.employments ?? [];
 
-  const personCristinIdentifier = getValueByKey('CristinIdentifier', personQuery.data?.identifiers);
   const topOrgCristinIdentifier = topOrgCristinId ? getIdentifierFromId(topOrgCristinId) : '';
+  const internalEmployments: Employment[] = [];
+  const externalEmployments: Employment[] = [];
+  const targetOrganizationIdStart = `${topOrgCristinIdentifier.split('.')[0]}.`;
```

Comment:
Dette er en litt skummel forenkling, da det ikke er gitt at alle har samme tall først i identifier. Men har vært sånn tidligere også uten at det har vært et problem for folk, så bryr meg ikke mer om det nå